### PR TITLE
Specs for ServiceTags resource

### PR DESCRIPTION
**Pull request by anton-evseev**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [x] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by number213** [comment]
cc: @chandrasekarsrinivasan

File: specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/serviceTags.json [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+      "get": {
+        "operationId": "ServiceTags_List",
+        "description": "Gets a list of service tag information resources.",
+        "parameters": [
+          {
+            "name": "location",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": ""
```
Comment (EvgenyAgafonchikov) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/serviceTags.json]: empty description

File: specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/serviceTags.json [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+          "Get list of service tags": {
+            "$ref": "./examples/ServiceTagsList.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ServiceTagsListResult": {
+      "properties": {
```
Comment (dsgouda) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/serviceTags.json]: This is a bit confusing.
The list result contains `name`, `id` and `type` along with an array of `ServiceTagInformation` objects which in turn have the same `name`, `id`, `type` properties?

**Review by dsgouda — CHANGES_REQUESTED** [review]

File: specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/serviceTags.json (hunk 2 above)
Comment (number213) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/serviceTags.json]: `name`. `id` in `ServiceTagsListResult` are for from which cloud the data is (Public, Chinese, US Government or German Government).
`name` and `id` of `ServiceTagInformation` show which service the inforrmation is about

```json
{
  "name": "public",
  "id": "/subscriptions/subId/providers/Microsoft.Network/serviceTags/public",
  "values": [
    {
      "name": "ApiManagement",
      "id": "ApiManagement"
     }
  ]
}
```

Maybe @chandrasekarsrinivasan could give more context if it's still unclear

Comment (dsgouda) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/serviceTags.json]: Thanks for the explanation. I think someone from ARM should sign this off.
Per your explanation it seems like name, id, type must be readonly

Comment (dsgouda) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/serviceTags.json]: Per offline thread, these are read only operations, please mark the appropriate properties as readonly

Comment (chandrasekarsrinivasan) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/serviceTags.json]: Already sent the ARM approval email.

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**